runtime/etc/init/planb-tokeninfo.conf

```diff
@@ -14,8 +14,20 @@ script
         export OPENID_PROVIDER_CONFIGURATION_URL="$config_openid_provider_configuration_url"
         export REVOCATION_PROVIDER_URL="$config_revocation_provider_url"
         export BUSINESS_PARTNERS="$config_tokeninfo_business_partners"
+        export OPENTRACING_LIGHTSTEP_ACCESS_TOKEN="$config_opentracing_lightstep_access_token"
+        export OPENTRACING_LIGHTSTEP_COMPONENT_NAME="$config_opentracing_lightstep_component_name"
+        export ARTIFACT="$config_artifact"
+        export OPENTRACING_LIGHTSTEP_COLLECTOR_HOST="$config_opentracing_lightstep_collector_host"
+        export OPENTRACING_LIGHTSTEP_COLLECTOR_PORT="$config_opentracing_lightstep_collector_port"
+        export OPENTRACING_MAX_BUFFERED_SPANS="$config_opentracing_max_buffered_spans"
+        export ACCOUNT="$config_account"
+        export ZONE="$config_zone"
+        export DEPLOYMENT_ID="$config_deployment_id"
```

Comment:
Not sure who's supposed to populate it. Senza?

Reply:
Ideally this should be the senza stack version. At the moment we expect the users to set this as an environment variable from their cloud formation yaml. 
Is there a way that I can access the stack version from here?